language_service/src/completion.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#![allow(warnings)]
```

Comment:
Maybe a comment why this is needed?

Reply:
This was actually a mistake, I meant to remove it. Whoops! Will fix.

Reply:
Fixed in cd458b68